pynvjitlink/tests/test_pynvjitlink_api.py

```diff
@@ -78,6 +78,7 @@ def test_add_fatbin_with_cubin_error(device_functions_cubin, gpu_arch_flag):
         nvjitlinker.add_fatbin(cubin, name)
 
 
+@pytest.mark.xfail(reason='CUDA 12.6 pynvjitlink does not raise for duplicates')
```

Comment:
To fix [the style check error]( https://github.com/rapidsai/pynvjitlink/actions/runs/11272141970/job/31346456395?pr=97#step:6:87 )

```suggestion
@pytest.mark.xfail(reason="CUDA 12.6 pynvjitlink does not raise for duplicates")
```